renderer/viewer/lib/worldrendererCommon.ts

```diff
@@ -36,6 +36,7 @@ export const defaultWorldRendererConfig = {
   mesherWorkers: 4,
   isPlayground: false,
   renderEars: true,
+  skinTexturesProxy: undefined as string | undefined,
```

Comment:
```suggestion
  skinPropertiesProxy: undefined as string | undefined,
```

i propose to change app config option name

something to make it clear that it only proxies skin loading only from server properties and does not replace builtin playerdb.co loading (based on username)

Reply:
Not too fond of that name seeing as it's the proxy for the `textures` property of a skull/player skin and the server is named `textures.minecraft.net` too. Your proposed name would make it really confusing what it really is referring to imo.

(Also generally speaking there should be an option/better approach to load player skins too. In the normal game the skin is always loaded from mojang directly, there should be no need to use an external service which isn't even mentioned anywhere so potentially even a privacy issue)

Reply:
I misunderstood your intention in the pr. Answered all this in the pr comment @Phoenix616 btw please feel free to ping me for faster response;)